lib/plugin.js

```diff
@@ -38,6 +38,7 @@ var createPreprocesor = function(logger, config, basePath) {
 
     var output =
       'window.__cjs_modules_root__ = "' + modulesRootPath + '";' +
+      (config.istanbul.ignore ? ' ' + config.istanbul.comment + ' ' : '') +
```

Comment:
Actually, the idea was not to make it required. If "config.istanbul.ignore" equals to undefined or false, then output will get an empty string. So, I do not need to specify any parameter in karma configuration if I do not use istanbul coverage.